index.html

```diff
@@ -51,18 +51,11 @@ <h1 class="welcome-title">Welcome to Real Dev Squad</h1>
           <h3>Check out our Docs</h3>
         </div>
         <div class="cards">
-          <div class="card">
-            <a href="./faq.html">FAQ</a>
-          </div>
-          <div class="card">
-            <a href="./discord.html">Discord</a>
-          </div>
-          <div class="card">
-            <a href="./code-of-conduct.html">Code of Conduct</a>
-          </div>
+          <a class="card" href="./faq.html">FAQ</a>
+          <a class="card" href="./discord.html">Discord</a>
+          <a class="card" href="./code-of-conduct.html">Code of Conduct</a>
```

Comment:
I guess we'll be good to go with links ending without `.html` 
What do you think?

Reply:
> I guess we'll be good to go with links ending without `.html`
> What do you think?
mmm sorry ! Not getting u